src/blocks/recommendations/components/parsely-recommendations-inspector-controls.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { InspectorControls } from '@wordpress/block-editor';
+import {
+	PanelBody,
+	PanelRow,
+	RadioControl,
+	RangeControl,
+	SelectControl,
+	TextControl,
+	ToggleControl,
+} from '@wordpress/components';
+
+const ParselyRecommendationsInspectorControls = ( {
+	attributes: { boost, imagestyle, layoutstyle, limit, showimages, sort, tag, title },
+	setAttributes,
+} ) => {
+	return (
+		<InspectorControls>
+			<PanelBody title="Settings" initialOpen={ true }>
+				<PanelRow>
+					<TextControl
+						label={ __( 'Title', 'wp-parsely' ) }
+						value={ title }
+						onChange={ ( newval ) => setAttributes( { title: newval } ) }
+					/>
+				</PanelRow>
+				<PanelRow>
+					<RangeControl
+						label={ __( 'Maximum Results', 'wp-parsely' ) }
+						min="1"
+						max="99"
+						onChange={ ( newval ) => setAttributes( { limit: newval } ) }
+						value={ limit }
+					/>
+				</PanelRow>
+				<PanelRow>
+					<ToggleControl
+						label={ __( 'Show Images', 'wp-parsely' ) }
+						help={
+							showimages
+								? __( 'Showing images', 'wp-parsely' )
+								: __( 'Not showing images', 'wp-parsely' )
+						}
+						checked={ showimages }
+						onChange={ () => setAttributes( { showimages: ! showimages } ) }
+					/>
+				</PanelRow>
+				{ showimages && (
+					<PanelRow>
+						<RadioControl
+							label={ __( 'Image style', 'wp-parsely' ) }
+							selected={ imagestyle }
+							options={ [
+								{ label: __( 'Original image', 'wp-parsely' ), value: 'original' },
+								{ label: __( 'Thumbnail from Parse.ly', 'wp-parsely' ), value: 'thumbnail' },
+							] }
+							onChange={ ( newval ) =>
+								setAttributes( {
+									imagestyle: newval === 'original' ? 'original' : 'thumbnail',
+								} )
+							}
+						/>
+					</PanelRow>
+				) }
```

Comment:
The images that we pull for this are of varying quality. Let's consider how to ensure that when images are enabled, we only show recommendations with a reasonably large image file.

Reply:
Now tracked in #703.